client/src/main/java/org/apache/cassandra/sidecar/client/SidecarClient.java

```diff
@@ -114,6 +115,22 @@ public CompletableFuture<HealthResponse> sidecarHealth()
                                             .build());
     }
 
+    /**
+     * Executes the Sidecar health request using the configured selection policy and with no retries
+     *
+     * @return a completable future of the Sidecar health response
+     */
+    public CompletableFuture<HealthResponse> peerHealth(SidecarInstance instance)
+    {
+        return executor.executeRequestAsync(requestBuilder()
+                                                .singleInstanceSelectionPolicy(
+                                                new SidecarInstanceImpl(instance.hostname(),
+                                                                        instance.port()))
+                                                .retryPolicy(oncePerInstanceRetryPolicy)
+                                                .sidecarHealthRequest()
+                                                .build());
```

Comment:
not aligned properly. 

```suggestion
        return executor.executeRequestAsync(requestBuilder()
                                            .singleInstanceSelectionPolicy(instance)
                                            .retryPolicy(oncePerInstanceRetryPolicy)
                                            .sidecarHealthRequest()
                                            .build());
```